simapp/v2/simdv2/cmd/commands.go

```diff
@@ -230,11 +207,5 @@ func appExport(
 }
 
 var tempDir = func() string {
```

Comment:
We should be able to kill this completely. This was a hack in server v0, so let's remove this